<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IsletmeVeriGirisFormu</class>
 <widget class="QWidget" name="IsletmeVeriGirisFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#btnKaydet {
	qproperty-icon: url(:/Resimler/Kaydet.png);
	qproperty-iconSize: 16px 16px;
   qproperty-text: &quot;&amp;Kaydet&quot;;
   qproperty-default: true;
}

QPushButton#btnIptal {
	qproperty-icon: url(:/Resimler/Iptal.png);
	qproperty-iconSize: 16px 16px;
   qproperty-text: &quot;İ&amp;ptal&quot;;
}

QLabel {
	font: 700 12pt &quot;Roboto&quot;;
}

QLabel#lblBaslik {
	font: 700 italic 18pt &quot;Roboto&quot;;
}

QLabel#lblSimge {
	qproperty-maximumSize: 48px 48px;
}

QLineEdit {
	font: 12pt &quot;Roboto&quot;;
}
QSpinBox {
	font: 12pt &quot;Roboto&quot;;
}

QPlainTextEdit {
	font: 12pt &quot;Roboto&quot;;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblSimge">
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>:/Resimler/YeniIsletme</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblBaslik">
         <property name="text">
          <string>Yeni İşletme Girişi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="Line" name="line">
       <property name="styleSheet">
        <string notr="true">QPushButton#btnKaydet {
	qproperty-icon: url(:/Resimler/Kaydet.png);
	qproperty-iconSize: 16px 16px;
   qproperty-text: &quot;&amp;Kaydet&quot;;
   qproperty-default: true;
}

QPushButton#btnIptal {
	qproperty-icon: url(:/Resimler/Iptal.png);
	qproperty-iconSize: 16px 16px;
   qproperty-text: &quot;İ&amp;ptal&quot;;
}

QLabel {
	font: 700 12pt &quot;Roboto&quot;;
}

QLabel#lblBaslik {
	font: 700 italic 18pt &quot;Roboto&quot;;
}

QLabel#lblSimge {
	qproperty-maximumSize: 48px 48px;
}

QLineEdit {
	font: 12pt &quot;Roboto&quot;;
}
QSpinBox {
	font: 12pt &quot;Roboto&quot;;
}

QPlainTextEdit {
	font: 12pt &quot;Roboto&quot;;
}</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QLineEdit" name="leKurucuAd">
           <property name="placeholderText">
            <string>Bir Ad giriniz</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="leYerlesmeYeri">
           <property name="placeholderText">
            <string>Bir yerleşim yeri giriniz</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblIsletmeAdi">
           <property name="text">
            <string>İşletme Adı</string>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="leVergiNumarasi">
           <property name="placeholderText">
            <string>Bir vergi numarası giriniz</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblKurucuAd">
           <property name="text">
            <string>Kurucu Adı</string>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="leKurucuSoyad">
           <property name="placeholderText">
            <string>Bir soyad giriniz</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblVergiNumarasi">
           <property name="text">
            <string>Vergi Numarası</string>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblKurucuSoyad">
           <property name="text">
            <string>Kurucu Soyadı</string>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblYerlesmeYeri">
           <property name="text">
            <string>Adres</string>
           </property>
           <property name="margin">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leIsletmeAdi">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
           <property name="placeholderText">
            <string>Bir Ad giriniz</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnKaydet">
           <property name="text">
            <string>&amp;Kaydet</string>
           </property>
           <property name="icon">
            <iconset resource="../../Kaynaklar.qrc">
             <normaloff>:/Resimler/Kaydet.png</normaloff>:/Resimler/Kaydet.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnIptal">
           <property name="text">
            <string>İ&amp;ptal</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/Resimler/IptalEt</normaloff>:/Resimler/IptalEt</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Kaynaklar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnIptal</sender>
   <signal>clicked()</signal>
   <receiver>IsletmeVeriGirisFormu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
